content/docs/user-guide/data-management/large-dataset-optimization.md

```diff
@@ -98,6 +98,11 @@ efficiency:
 
 [replace them]: /doc/user-guide/how-to/update-tracked-files
 
+<admon type="tip">   
+Use `dvc version` in a DVC repository (with an existing cache folder) to display
```

Comment:
btw, @yathomasi - for whatever reason the 80 symbols rule was not triggered within this admon section. Not critical at all. If you know from the top your had what it is about we can fix it.

Reply:
One caveat is the multiline snippet inside a `tag` also needs to be wrapped with newlines for formatting to work properly.

Eg:
```html
<admon type="tip">   

content

</admon>
```